database/seed.js

```diff
@@ -46,6 +46,15 @@ let createSummary = function() {
   }
 };
 
+let retrieve = (callback) => {
```

Comment:
I would suggest you be more specific with the naming of your functions. I'm not sure what data this is retrieving exactly. It looks like it's retrieving everything from `itemList` but I'm not sure what the data structure of itemList is, and looking at your server file below, it looks like you're hitting a `/shop` endpoint to call on this `retrieve` function. I think you are trying to get all items for a given shop. Perhaps a better name for this function would be something like `retrieveShopItems`.